### PR TITLE
Add FAQ about overriding the product cover

### PR DESCRIPTION
**Pull request by jolelievre**
In 177 the cover image is used by default in product listing, this explains how to verride this behaviour starting .7.7

### CODE REVIEW

**Review by Matt75 — APPROVED** [review]

File: src/content/1.7/faq/product.md
Comment (PierreRambaud) [inline src/content/1.7/faq/product.md:14]: It's not really a property :thinking: 
`$product` is an array, not an object

Comment (Progi1984) [reply src/content/1.7/faq/product.md:14]:
```suggestion
**A:** By default, when a product is displayed in a list its cover image is used, it is configurable in the BackOffice and is set in product properties via the `cover_image_id` key.
```

**Review by Progi1984 — COMMENTED** [review]
Ping @jolelievre

File: src/content/1.7/faq/product.md
Comment (jolelievre) [reply src/content/1.7/faq/product.md]: I used the term `property` because it was in the context of the `getProductProperties` function, besides `$product` is an array at the beginning of the workflow and is then converted to a `ProductLazyArray` which is a mix between an array and an object 😅

But I see your point and I agree so to avoid any misunderstanding I used the term `key` instead as suggested

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
We should wait for the original pull request to be merged before merging this one :)

**Comment by jolelievre** [comment]
Pending core PR https://github.com/PrestaShop/PrestaShop/pull/19437

**Pull request merged**
